plugin/pkg/admission/noderestriction/admission_test.go

```diff
@@ -376,6 +386,22 @@ func Test_nodePlugin_Admit(t *testing.T) {
 	pvcpod, _ := makeTestPod("ns", "mypvcpod", "mynode", true)
 	pvcpod.Spec.Volumes = []api.Volume{{VolumeSource: api.VolumeSource{PersistentVolumeClaim: &api.PersistentVolumeClaimVolumeSource{ClaimName: "foo"}}}}
 
+	orphanedPod, _ := makeTestPod("ns", "annie", "mynode", true)
+	orphanedPod.OwnerReferences = nil
+
+	invalidOwnerPod, _ := makeTestPod("ns", "alice", "mynode", true)
+	controller := true
+	invalidOwnerPod.OwnerReferences[0] = metav1.OwnerReference{
```

Comment:
test the various permutations of invalid owner refs:
* more than one
* invalid node name
* invalid node uid
* blocksOwnerDeletion: &true
* controller: nil
* controller: &false